datasets/tne/README.md

```diff
@@ -0,0 +1,186 @@
+  * `end_token`: an integer indicating the last token of the np out of the tokenized tokens
+  * `id`: the id of the np
+* `np_relations`: these are the relation labels of the document. It is a list of dictionaries, where each
+dictionary contains:
+  * `anchor`: the id of the anchor np
+  * `complement`: the id of the complement np
+  * `preposition`: the preposition that links between the anchor and the complement
+  * `complement_coref_cluster_id`: the coreference id, which the complement is part of.
+* `coref`: the coreference labels. It contains a list of dictionaries, where each dictionary contains:
+  * `id`: the id of the coreference cluster
+  * `members`: the ids of the nps members of such cluster
+  * `np_type`: the type of cluster. It can be either 
+    * `standard`: regular coreference cluster
+    * `time/date/measurement`: a time / date / measurement np. These will be singletons.
+    * `idiomatic`: an idiomatic expression
+* `metadata`: metadata of the document. It contains the following:
+  * `annotators`: a dictionary with anonymized annotators id
+    * `coref_worker`: the coreference worker id
+    * `consolidator_worker`: the consolidator worker id
+    * `np-relations_worker`: the np relations worker id
+  * `url`: the url where the document was taken from (not always existing)
+  * `source`: the original file name where the document was taken from
+
+
+### Data Splits
+
+The dataset is spread across four files, for the four different splits: train, dev, test and ood.
+Additional details on the data statistics can be found in the [paper](https://arxiv.org/abs/2109.12085)
+
+## Dataset Creation
+
+### Curation Rationale
+
+TNE was build as a new task for language understanding, focusing on extracting relations between nouns, moderated by prepositions.
+
+### Source Data
+
+#### Initial Data Collection and Normalization
+
+[Needs More Information]
+
+#### Who are the source language producers?
+
+[Needs More Information]
+
+### Annotations
+
+#### Annotation process
+
+[Needs More Information]
+
+#### Who are the annotators?
+
+[Needs More Information]
+
+### Personal and Sensitive Information
+
+[Needs More Information]
+
+## Considerations for Using the Data
+
+### Social Impact of Dataset
+
+[Needs More Information]
+
+### Discussion of Biases
+
+[Needs More Information]
+
+### Other Known Limitations
+
+[Needs More Information]
+
+## Additional Information
+
+### Dataset Curators
+
+The dataset was created by Yanai Elazar, Victoria Basmov, Yoav Goldberg, Reut Tsarfaty, during work done at Bar-Ilan University, and AI2.
+
+### Licensing Information
+
+The data is released under the MIT license.
+
+### Citation Information
+
+@article{tne,
+    author = {Elazar, Yanai and Basmov, Victoria and Goldberg, Yoav and Tsarfaty, Reut},
+    title = "{Text-based NP Enrichment}",
+    journal = {Transactions of the Association for Computational Linguistics},
+    year = {2022},
+}
```

Comment:
````suggestion
```bibtex
@article{tne,
    author = {Elazar, Yanai and Basmov, Victoria and Goldberg, Yoav and Tsarfaty, Reut},
    title = "{Text-based NP Enrichment}",
    journal = {Transactions of the Association for Computational Linguistics},
    year = {2022},
}
```
````